### PR TITLE
Update workers bbe for @concurrent annotation

### PR DESCRIPTION
**Pull request by vinok88**
Fixes #<Issue Number>

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: examples/workers/workers.bal [hunk 1]
```diff
@@ -19,6 +19,10 @@ public function main() {
         io:println("sum of first ", n, " positive numbers = ", sum);
     }
 
+    // By default workers run on the same physical thread of the parent strand.
```
Comment (praneesha) [inline examples/workers/workers.bal]:
```suggestion
    // By default, workers run on the same physical thread of the parent strand.
```

File: examples/workers/workers.description [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Workers in Ballerina allow developers to delegate their tasks to parallel running executions.
+// Workers in Ballerina allow developers to delegate their tasks to independently running executions.
```
Comment (praneesha) [inline examples/workers/workers.description]:
```suggestion
// Workers in Ballerina allow developers to delegate their tasks to independently-running executions.
```

File: examples/async/async.bal [hunk 3]
```diff
@@ -25,7 +25,9 @@ public function main() {
     io:println("Counting done in one second: ", count);
 
     // Asynchronously invokes the action call `get()`.
-    future<http:Response | error> f3 = start clientEndpoint-> get("/get?test=123");
+    // By default, this async call runs on the same physical thread of the caller.
+    // @concurrent{} annotation allows the invocation to run parallel.
```
Comment (MaryamZi) [inline examples/async/async.bal]:
```suggestion
    // The `@concurrent{}` annotation allows the invocation to run parallel.
```

Also should "run parallel" be "run in parallel"?

File: examples/async/async.bal [hunk 4]
```diff
@@ -25,7 +25,9 @@ public function main() {
...
+    future<http:Response | error> f3 = @concurrent{} start clientEndpoint-> get("/get?test=123");
```
Comment (MaryamZi) [inline examples/async/async.bal]:
```suggestion
    future<http:Response | error> f3 = @concurrent{} start clientEndpoint->get("/get?test=123");
```

File: examples/workers/workers.bal [hunk 5]
```diff
@@ -19,6 +19,10 @@ public function main() {
         io:println("sum of first ", n, " positive numbers = ", sum);
     }
 
+    // By default, workers run on the same physical thread of the parent strand.
+    // By adding @concurrent annotation, this worker's strand is allowed to run
```
Comment (MaryamZi) [inline examples/workers/workers.bal]:
```suggestion
    // By adding the `@concurrent` annotation, this worker's strand is allowed to run
```

Comment (MaryamZi) [reply examples/workers/workers.bal]: This doesn't have `{}` for the annotation btw. Shouldn't this be consistent with https://github.com/ballerina-platform/ballerina-lang/pull/20350/files#diff-8ab9d08acec1364f962b1a776e9cec9fR29?

**Pull request closed without merging**